docs/huggingface/tgi-0.9.3.md

```diff
@@ -0,0 +1,42 @@
+| MAX_STOP_SEQUENCES       |              4              | This is the maximum allowed value for clients to set `stop_sequences`. Stop sequences are used to allow the model to stop on more than just the EOS token, and enable more complex "prompting" where users can preprompt the model in a specific way and define their "own" stop token aligned with their prompt                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| MAX_INPUT_LENGTH         |            1024             | This is the maximum allowed input length (expressed in number of tokens) for users. The larger this value, the longer prompt users can send which can impact the overall memory required to handle the load. Please note that some models have a finite range of sequence they can handle                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| MAX_TOTAL_TOKENS         |            2048             | This is the most important value to set as it defines the "memory budget" of running clients requests. Clients will send input sequences and ask to generate `max_new_tokens` on top. with a value of `1512` users can send either a prompt of `1000` and ask for `512` new tokens, or send a prompt of `1` and ask for `1511` max_new_tokens. The larger this value, the larger amount each request will be in your RAM and the less effective batching can be                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
Needs update of casing (ex. `top. with a value...` should be `top. With a value...`